app/src/main/java/nodomain/freeyourgadget/gadgetbridge/service/devices/miband/MiBandSupport.java

```diff
@@ -276,7 +279,7 @@ private MiBandSupport setFitnessGoal(TransactionBuilder transaction) {
             transaction.write(characteristic, new byte[]{
                     MiBandService.COMMAND_SET_FITNESS_GOAL,
                     0,
-                    (byte) (fitnessGoal  &  0xff),
+                    (byte) (fitnessGoal & 0xff),
```

Comment:
It's just the automatic code style fix performed by android studio (I did run the tool before committing).